components/viewer/modal/ViewBackpack.vue

```diff
@@ -34,10 +34,10 @@
           <textarea class="export-code form-control" :value="exportCode" />
           <button
             class="export-btn btn btn-outline-primary"
+            :class="{ isCopied: isCopied }"
             @click="copyExportCode"
-          >
-            Copy to Clipboard
-          </button>
+            v-text="clipboardText"
+          ></button>
```

Comment:
It would be better to have two `<span>` elements inside that are displayed depending on the `isCopied` class 

This way it's easier to style them and show icons. 🙂